prow/config.yaml

```diff
@@ -392,6 +392,26 @@ presubmits:
       - name: docker-graph
         emptyDir: {}
 
+  kubernetes/cloud-provider-openstack:
+  - name: pull-cloud-provider-openstack-test
+    agent: kubernetes
+    always_run: true
+    context: pull-cloud-provider-openstack-test
+    rerun_command: "/test pull-cloud-provider-openstack-test"
+    trigger: "(?m)^/test( all| pull-cloud-provider-openstack-test),?(\\s+|$)"
+    branches:
+    - master
+    labels:
+      preset-service-account: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
```

Comment:
it *might* be better to use kubekins-e2e to match a kubernetes release branch's go version.

Reply:
warning: this image is pinned to gcloud 163.0.0 :-)

Reply:
ack. switching to `gcr.io/k8s-testimages/kubekins-e2e:v20180316-92e6ccade-master`